### ISSUE TITLE
Is Visual Paradigm Supported by Jhipster-uml?

### ISSUE THREAD

**Issue by verrol** [jhipster/jhipster-uml#250]
### The issue
I created a .xmi file from Visual Paradigm, but when I ran JHipster-UML on the file, it failed. The only editor option was 'umldesigner'.

---
### The command

$ jhipster-uml salepoint.xmi 
---
#### The stacktrace

```
An error has occurred:
	TypeError
Error message:
	Cannot read property 'length' of undefined
Stack trace:
TypeError: Cannot read property 'length' of undefined
    at findElementsInNode (/Users/another/.nvm/versions/node/v7.4.0/lib/node_modules/jhipster-uml/lib/editors/umldesigner_parser.js:56:43)
    at findElements (/Users/another/.nvm/versions/node/v7.4.0/lib/node_modules/jhipster-uml/lib/editors/umldesigner_parser.js:52:3)
    at Object.parse (/Users/another/.nvm/versions/node/v7.4.0/lib/node_modules/jhipster-uml/lib/editors/umldesigner_parser.js:25:3)
    at Object.<anonymous> (/Users/another/.nvm/versions/node/v7.4.0/lib/node_modules/jhipster-uml/lib/jhipsteruml.js:54:25)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)

```

---

### Steps to reproduce

1. Create a few tables and link them in Visual Paradigm
2. Export design to XMI
3. Run jhipster-ulm with this .xmi file

---

### The XMI file

---

### Versions 

| JHipster           | JHipster UML           |
| ------------------ | ---------------------- |
| 4.0.2 |    2.0.3  |

[sale-point.xmi.zip](https://github.com/jhipster/jhipster-uml/files/769423/sale-point.xmi.zip)

**Comment by jdubois** [comment]
The list of supported editors is in the introduction at: https://jhipster.github.io/jhipster-uml/

So no, we don't support this one.

**Comment by jdubois** [comment]
As Visual Paradigm is proprietary software, you should first ask your software vendor about their support of the XMI format.

**Comment by MathieuAA** [comment]
Hi. It was supported but I decided to drop the support as it's proprietary and a pain in the a*s to develop a parser for the XMI output.

Le 12 février 2017 17:18:25 GMT+01:00, Julien Dubois <notifications@github.com> a écrit :
>Closed #250.
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/jhipster/jhipster-uml/issues/250#event-958694282

-- 
Mathieu ABOU-AICHI
